ipamd/datastore/data_store.go

```diff
@@ -26,34 +26,39 @@ import (
 const (
 	minLifeTime          = 1 * time.Minute
 	addressCoolingPeriod = 1 * time.Minute
-	// DuplicatedENIError is an error when caller tries to add an duplicate eni to data store
-	DuplicatedENIError = "data store: duplicate eni"
+	// DuplicatedENIError is an error when caller tries to add an duplicate ENI to data store
+	DuplicatedENIError = "data store: duplicate ENI"
 
-	// DuplicateIPError is an error when caller tries to add an duplicate ip address to data store
+	// DuplicateIPError is an error when caller tries to add an duplicate IP address to data store
 	DuplicateIPError = "datastore: duplicated IP"
 )
 
 // ErrUnknownPod is an error when there is no pod in data store matching pod name, namespace, container id
 var ErrUnknownPod = errors.New("datastore: unknown pod")
 
-// ErrUnknownPodIP is an error where pod's ip address is not found in data store
-var ErrUnknownPodIP = errors.New("datastore: pod using unknown ip address")
+// ErrUnknownPodIP is an error where pod's IP address is not found in data store
+var ErrUnknownPodIP = errors.New("datastore: pod using unknown IP address")
 
-// ENIIPPool contains ENI/IP Pool information
+// ENIIPPool contains ENI/IP Pool information. The capitalized variables are used for introspection.
```

Comment:
s/The capitalized variables are used for introspection/Exported fields will be Marshaled for introspection/